boilerplate/dap/status-CRD.include

```diff
@@ -46,7 +46,7 @@
   is to have a minimum of two independent and interoperable user agents that
   implementation all the features of this specification, which will be determined by
```

Comment:
This text inherited from the 3-year old boilerplate "user agents that implementation all the features" did not sound right to me. How about "user agents that implement all the features". Patch:

```suggestion
  implement all the features of this specification, which will be determined by
```

Reply:
updated two (CR/CRD) files